components/hook-fixture/fixture/index.mjs

```diff
@@ -37,7 +37,11 @@ export const testHookAsync = async (
   try {
     recordStartTrack(agent, "record", {}, null);
     await callbackAsync();
-    recordStopTrack(agent, "record", { type: "manual" });
+    // TODO this completely breaks encapsulation
+    //      but hook-fixture is only used for testing...
```

Comment:
I think breaking encapsulation for tests is acceptable sometimes.

Reply:
done better than perfect?

Reply:
Yup :)